examples/vm_snapshot_removal_based_on_timestamp.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Example - delete all snapshots that are older than X days.
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  environment:
+    SC_HOST: "your_host"
+    SC_USERNAME: "your_username"
+    SC_PASSWORD: "your_password"
+  vars:
+    # Format: 'YYYY-MM-DD hh:mm:ss'
+    # All snapshots older than this date will be deleted.
+    use_date: '2023-05-03 12:52:00'
```

Comment:
Change default `use_date` to some "safer" value. If I run example unintentionally, I will delete snapshots, and that can be a problem. Something like `1999-05-03 12:52:00`?